src/components/LandingPage.jsx

```diff
@@ -1,13 +1,17 @@
-import React from 'react';
-
+import React, { useState, useEffect } from 'react';
+import requestHandler from '../services/requestHandler'
 
 const LandingPage = () => {
+  const [landingPageContent, setlandingPageContent] = useState('');
 
+  useEffect(() => {
+    requestHandler.homeUrl().then(res => setlandingPageContent(res.data.message))
```

Comment:
I am thinking that since the `landingPageContent` is static, it might be expensive making an api call just to get it every time we navigate to that page. The expensiveness will be even more pronounced when making a call to the heroku hosted app which is much slower than localHost.

For this reason, we might need to just revert this page to what it was before.